runtime/nls/j9vm/j9vm.nls

```diff
@@ -2097,3 +2097,11 @@ J9NLS_ENSUREHASHED_CONFIG_OUT_OF_MEM.explanation=There was a failure to allocate
 J9NLS_ENSUREHASHED_CONFIG_OUT_OF_MEM.system_action=The JVM prints this message and terminates.
 J9NLS_ENSUREHASHED_CONFIG_OUT_OF_MEM.user_response=Check the system memory configuration.
 # END NON-TRANSLATABLE
+
+# JVMCheckpointException is thrown to indicate that blocking operation is not allowed at CRIU single thread mode
```

Comment:
the messaging should be generic enough such that it can be displayed on the checkpoint or restore side